projects/cashmere-examples/src/lib/navbar-overview/navbar-overview-example.component.html

```diff
@@ -9,7 +9,10 @@
     <hc-icon class="hc-navbar-icon" fontSet="fa" fontIcon="fa-question-circle-o" [hcPop]="options"></hc-icon>
     <span class="hc-navbar-vertical-separator"></span>
     <div class="hc-navbar-username" *ngIf="user" [hcPop]="user">
-        <span class="hc-text-ellipsis">{{username}}</span>
+        <span>
+            <span>{{username}}</span><br>
+            <span style="color:#c0c5cc">{{organization}}</span>
```

Comment:
Looks great! My only suggestion would be to put this style in the stylesheet.

Reply:
Great suggestion @Br00t4L17y - I hate hard-coding color values into the examples anyway.  Just pushed a commit that adds a `hc-navbar-username-subtext` class.